service/src/intTest/resources/application.yml

```diff
@@ -14,6 +14,8 @@ management:
         include: info, health, metrics
 
 gp2gp:
+  trustStoreUrl: ${GP2GP_SSL_TRUST_STORE_URL:}
+  trustStorePassword: ${GP2GP_SSL_TRUST_STORE_PASSWORD:}
```

Comment:
This could be moved to storage?

Reply:
Storage is more about storing files. When I took a look at nhais service it had this data constructed. like this.

Reply:
Yes this refers to the Java Trust Store where CA certificate chains are held. Applies to the entire adaptor.